app/src/main/java/de/westnordost/streetcomplete/screens/main/map/QuestPinsManager.kt

```diff
@@ -111,7 +111,7 @@ class QuestPinsManager(
     fun onNewScreenPosition() {
         if (!isActive) return
         val zoom = ctrl.cameraPosition.zoom
-        if (zoom < TILES_ZOOM) return
+        if (zoom < 14) return
```

Comment:
Wouldn't another constant rather than a magic number be a better idea? Then the same value could potentially be used in the other file too?

Reply:
🤷 or a comment